packages/neuron-ui/src/components/WalletWizard/hooks.ts

```diff
@@ -7,6 +7,17 @@ export const useInputWords = () => {
       const idx = Number(e.target.dataset.idx)
       if (Number.isNaN(idx)) return
       const { value } = e.target
+      if (Number(idx) === 0) {
+        const list = value
+          .trim()
+          .replace(/[^0-9a-z]+/g, ' ')
+          .split(' ')
+        if (list.length === 12) {
+          setInputsWords(list)
+          return
+        }
+      }
```

Comment:
I do not comprehend the function of this segment of code.

Reply:
> I do not comprehend the function of this segment of code.

<img width="734" alt="image" src="https://github.com/nervosnetwork/neuron/assets/11756869/36ec2f27-1fc2-4ebe-8477-a8a569e93453">

Reply:
@Danie0918 Maybe we should add more tips to tell the user that the first input can enter all of the seed phrases

Reply:
> @Danie0918 Maybe we should add more tips to tell the user that the first input can enter all of the seed phrases

UI：https://www.figma.com/design/6XNoimRDbFTTNm016rbIdU/Magickbase?node-id=319-4539&t=a0LYiUc0DrgWeXgO-0

Added. @devchenyan